index.html

```diff
@@ -0,0 +1,762 @@
+                  </div>
+                </div>
+                <div>
+                  <p class="grid-image-title">
+                    Comet NEOWISE: How to See It in Night Skies
+                  </p>
+                  <p class="grid-image-date">
+                    July 15
+                  </p>
+                </div>
+              </a>
+            </div>
+            <div class="bottom-grid-division">
+            </div>
+            <div>
+              <h3>
+                Editor's Picks
+              </h3>
+            </div>
+            <div class="bottom-grid">
+              <a href="#">
+                <div>
+                  <img src="images/bot-grid-1.jpg" alt="Bottom grid 1st image">
+                  <div>
+                    John Taggart for The New York Times
+                  </div>
+                </div>
+                <div>
+                  <p class="grid-image-title">
+                    Scientists Have Finally Calculated How Many Hot Dogs a Person Can Eat at Once
+                  </p>
+                  <p class="grid-image-date">
+                    July 14
+                  </p>
+                </div>
+              </a>
+              <a href="#">
+                <div>
+                  <img src="images/bot-grid-2.jpg" alt="Bottom grid 2nd image">
+                  <div>
+                    Akasha Rabut for The New York Times
+                  </div>
+                </div>
+                <div>
+                  <p class="grid-image-title">
+                    He Kept a Community’s Bears in Check, but He Won’t Anymore
+                  </p>
+                  <p class="grid-image-date">
+                    July 20
+                  </p>
+                </div>
+              </a>
+              <a href="#">
+                <div>
+                  <img src="images/bot-grid-3.jpg" alt="Bottom grid 3rd image">
+                  <div>
+                    Meridith Kohut for The New York Times
+                  </div>
+                </div>
+                <div>
+                  <p class="grid-image-title">
+                    The Great Climate Migration Has Begun
+                  </p>
+                  <p class="grid-image-date">
+                    45m ago
+                  </p>
+                </div>
+              </a>
+            </div>
+          </div>
+          <div class="most-popular">
+            <div>
+              <h3>
+                Most Popular
+              </h3>
+            </div>
+            <div>
+              <p>
+                Judge Orders Cohen Released, Citing ‘Retaliation’ Over Tell-All Book
+              </p>
+            </div>
+            <div>
+              <p>
+                When Brad Photographed Gwyneth
+              </p>
+            </div>
+            <div>
+              <p>
+                Trump Defends His Cognitive Testing Results on Fox News. Again.
+              </p>
+            </div>
```

Comment:
## Bottom part

It contains links to more popular articles on the rightmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

- [ ] All articles on the most-popular class should have `<a href="#">` tag